nipype/interfaces/ants/preprocess.py

```diff
@@ -330,12 +332,13 @@ def _run_interface(self, runtime):
         next(in_data)
 
         for x in in_data:
-            t1 = math.atan2(float(x[7]), float(x[10]))
-            c2 = math.sqrt((float(x[2]) * float(x[2])) + (float(x[3]) * float(x[3])))
-            t2 = math.atan2(-float(x[4]), c2)
-            t3 = math.atan2(float(x[3]), float(x[2]))
+            mat = numpy.zeroes((3, 3))
```

Comment:
why don't loading data with numpy.loadtxt, and then use numpy fancy indexing for all of this?